src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -14,7 +14,6 @@ public static class MetricEventNames
         public const string HostStartupInitializeBindingProvidersLatency = "host.startup.initializebindingproviders.latency";
         public const string HostStartupCreateMetadataProviderLatency = "host.startup.createmetadataprovider.latency";
         public const string HostStartupGetFunctionDescriptorsLatency = "host.startup.getfunctiondescriptors.latency";
-        public const string HostStartupPurgeLogDirectoriesLatency = "host.startup.purgelogdirectories.latency";
```

Comment:
Now spinning up the purge operation on a background task, so it isn't a synchronous factor of the startup path anymore